external-wallets/wagmi-wallets/src/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const WALLETS = [
+  {
+    type: "metamask",
+    name: "MetaMask",
+  },
```

Comment:
Whenever I attempt to connect the MetaMask wallet, the wallet pop-up appears. Is there a way to prevent this from happening?

Reply:
Based on its playground and all websites that already use Wagmi, it always opens a popup for MetaMask (probably to handle multiple accounts).